start-site/src/main/resources/application.yml

```diff
@@ -65,21 +65,21 @@ initializr:
         versionProperty: sentry.version
         mappings:
           - compatibilityRange: "[3.2.0,3.3.0-M1)"
-            version: 7.3.0
+            version: 7.14.0
       solace-spring-boot:
         groupId: com.solace.spring.boot
         artifactId: solace-spring-boot-bom
         versionProperty: solace-spring-boot.version
         mappings:
           - compatibilityRange: "[3.2.0,3.4.0-M1)"
-            version: 2.0.0
+            version: 2.1.1
       solace-spring-cloud:
         groupId: com.solace.spring.cloud
         artifactId: solace-spring-cloud-bom
         versionProperty: solace-spring-cloud.version
         mappings:
           - compatibilityRange: "[3.2.0,3.4.0-M1)"
-            version: 4.1.0
+            version: 4.2.0
```

Comment:
We can't do that for all supported Boot versions (3.2.0, 3.3.0 and 3.4.0) - Solace 4.2.0 is [compatible with 3.3.x+ only](https://github.com/SolaceProducts/solace-spring-cloud/tree/4.4.0/solace-spring-cloud-bom#spring-cloud-version-compatibility).

We need to a add a mapping:

- Boot 3.2.0 uses Solace 4.1.0
- Boot 3.3.x and 3.4.x uses Solace 4.4.0

Btw, is there a reason you only bumped it to 4.2.0 instead of 4.4.0?

Reply:
Updated... 
I put 4.2.0 because I didn't check the latest version recently and in my current job we are using Spring Boot 3.3.1 + Solace Spring Cloud 4.2.0
Thanks for your comments. It helped me a lot!
Have a great Friday!